cli/shared/src/main/scala/com/lightbend/rp/reactivecli/Main.scala

```diff
@@ -191,6 +191,13 @@ object Main extends LazyLogging {
                   }
                 }
 
+                def getDockerCmdConfig(imageName: String): Future[Option[Config]] =
+                  Platform.processExec(Seq("docker", "inspect", imageName)).map {
```

Comment:
Executing docker command should be wrapped into a separate object with debug logging, like other native tools that cli uses - jq, kubectl, docker-credentials.

Reply:
Will do.